channeldb/migrations.go

```diff
@@ -610,3 +611,63 @@ func migrateOptionalChannelCloseSummaryFields(tx *bbolt.Tx) error {
 
 	return nil
 }
+
+// migrateGossipMessageStoreKeys migrates the key format for gossip messages
```

Comment:
We can avoid the migration by interpreting the key `pubkey+shortchanid+<empty>` as `AnnounceSignatures`.

Reply:
I'm in favor of avoiding the migration (FWIW, it is a very simple one and includes tests) -- just went with it as it was pointed out in review that we should keep the format consistent. Will wait to let the others chime in here cc @Roasbeef @cfromknecht.